src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -62,14 +86,15 @@
     <Compile Include="System\Text\Json\Serialization\Converters\JsonValueConverterDateTimeOffset.cs" />
     <Compile Include="System\Text\Json\Serialization\Converters\JsonValueConverterDecimal.cs" />
     <Compile Include="System\Text\Json\Serialization\Converters\JsonValueConverterDouble.cs" />
-    <Compile Include="System\Text\Json\Serialization\Converters\JsonConverterEnum.cs" />
```

Comment:
Thanks for fixing up the ordering :)

Reply:
BTW we should fix the csproj to use the default sorting by VS. I think the easiest would be to remove and add all files.